src/IrcFeatures/DccChat.cs

```diff
@@ -130,8 +130,8 @@ internal override void InitWork(Object stateInfo)
 
             DccChatStartEvent(new DccEventArgs(this));
 
-            _sr = new StreamReader(Connection.GetStream(), Irc.Encoding);
-            _sw = new StreamWriter(Connection.GetStream(), Irc.Encoding);
+            _sr = new StreamReader(Connection.GetStream(), System.Text.Encoding.Default);
+            _sw = new StreamWriter(Connection.GetStream(), System.Text.Encoding.Default);
```

Comment:
That was just a hack to make it compile, I don't know how to resolve it. 'Irc' here is an IrcFeatures which has no Encoding property. The IrcConnection is never passed to DccChat. How did you get it to compile?